utility/utils.py

```diff
@@ -1446,6 +1446,9 @@ def run_fio(**fio_args):
     else:
         opt_args = f"--ioengine=rbd --rbdname={fio_args['image_name']} --pool={fio_args['pool_name']}"
 
+    if fio_args.get("size"):
+        opt_args += f" --size={fio_args.get('size')}"
+
```

Comment:
Can you please check line 1441
Can we please optimise usage of size in this method ?

Reply:
@VasishtaShastry But in my case I am not passing any filename all i want to specify is the size of IOs to be written, how do you think we can optimise this? is file_size a required option when file_name is specified or can it be optional?

Reply:
@manasagowri I think its required, I'm not sure for now
But It has been defaulted to 100M for now

Why can't we just remove `--size={fio_args.get('file_size', '100M')}` from 1441 and update method doc
(And see if anywhere in yamls or methods we are usig file_size and change it to size)